api.js

```diff
@@ -0,0 +1,78 @@
+// API written in express by charmines#1522
+exports.run = async (client) => {
+	const settings = require("./settings.json")
+	const bodyParser = require("body-parser");
+	const moment = require('moment')
+	const erl = require('express-rate-limit')
+	const express = require('express')
+	let app = express()
+
+	// Listen on port in settings
+	app.listen(settings.APIPort, (err) => {
+		if (err) {
+			console.log(err)
+		} else {
+			console.log('Ready at port ' + settings.APIPort)
+		}
+	})
+
+	// Handle requests
+	app.use(bodyParser.json({ type: "application/json" }));
+	app.use(express.urlencoded()) // Decode URLs
```

Comment:
```suggestion
	app.use(express.urlencoded()) // Decode application/x-www-form-urlencoded POST bodies
```

urlencoded doesn't actually decode URLs, it decodes `application/x-www-form-urlencoded` POST bodies.